h2o-core/src/main/java/water/parser/ParseDataset.java

```diff
@@ -1127,11 +1127,16 @@ public static void logParseResults(Frame fr) {
         }
       }
 
-      if (printLogSeparatorToStdout)
+      if (printLogSeparatorToStdout) {
         Log.info("Additional column information only sent to log file...");
```

Comment:
At some places you use LOG from log4j and at some places Log from water.util. Is it on purpose?

Reply:
it is impossible to replace all uses of water.Log in one PR, this has to be done gradually, and for some places around h2o-core we might stick to water.Log because of its semantics (handling newlines for example)